### DISCUSSION TITLE
缩略图是否可以存放到单独的目录

### DISCUSSION THREAD

**Discussion by qdljerry** [cloudreve/cloudreve#1758] [Q&A]
缩略图是否可以存放到单独的目录而不是源文件所在处

**Comment by YUDONGLING** [comment]
Support since 4.10.0.
Release note: [english](https://github.com/cloudreve/cloudreve/discussions/2169?sort=new#discussioncomment-14966746) and [chinese (简体中文)](https://github.com/cloudreve/cloudreve/discussions/2170?sort=new#discussioncomment-14966713)
